### DISCUSSION TITLE
[TOPAS validation]: Is simulating the boron proton fusion reaction possible in Topas?

### DISCUSSION THREAD

**Discussion by AIbrahimi2** [OpenTOPAS/OpenTOPAS#76] [Run-Problem] [answered]
### Contact Details

atia_ibrahimi@hotmail.com

### What was the issue?

I am trying to simulate the following reaction:
11B + 𝑝 → 12C → 8Be + 𝛼 → 3𝛼 + 8.7 MeV

I shoot protons at a B target and expect to measure the produced alphas. The maximum cross section of this reaction should be at a proton energy of about 500/600 keV, however at these energies I don't detect any alpha particles.

Has anyone been able to achieve this? Or does anyone know if it's possible in Topas and if so, if I should be using a specific physics list?

Thank you in advance!

### TOPAS category

Physics

### Version

3.7

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment, marked as answer]
Hi there,
The module `g4h-phy_QGSP_BIC_AllHP` grants you access to the TENDL database which has reaction cross-sections for protons with several materials. I am not sure if the explicit chain you describe is there. This is the reference for the [TENDL data base](https://www.sciencedirect.com/science/article/pii/S009037521930002X). Also check the readme files in `$HOME/Applications/GEANT4/G4DATA/G4TENDL1.4`.
Good luck.

**Reply by AIbrahimi2** [reply]
Dear Jose,

Thank you so much. I will try it out and see if it helps.
